src/main/java/com/hackathonorganizer/hackathonwriteservice/team/model/Team.java

```diff
@@ -0,0 +1,17 @@
+package com.hackathonorganizer.hackathonwriteservice.team.model;
+
+import javax.persistence.*;
+
+@Entity
+public class Team {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "hackathon_id")
+    private Long hackathonId;
```

Comment:
tutaj poproszę annotacje `@NotNull/@NotEmpty` wszędzie gdzie jest nullable=false żeby to na etapie walidacji hibernate wywalać a nie dopiero na zwrotce z bazy, to samo w innych encjach